schema.sql

```diff
@@ -0,0 +1,16 @@
+-- Database creation
+CREATE DATABASE vet_clinic;
+
+-- Table: public.animals
+
+-- DROP TABLE IF EXISTS public.animals;
+
+CREATE TABLE IF NOT EXISTS public.animals
+(
+    id integer,
+    name character varying(50) COLLATE pg_catalog."default",
+    date_of_birth date,
+    escape_attempts integer,
+    neutered boolean,
+    weight_kg double precision
+);
```

Comment:
- [ ] [Optional] It would be better professionally if you make animals table like that:
```sql
CREATE TABLE IF NOT EXISTS animals (
    id int GENERATED ALWAYS AS IDENTITY,
    name varchar(255),
    date_of_birth date,
    escape_attempts int,
    neutered boolean,
    weight_kg decimal, 
 );
```